node/index.js

```diff
@@ -461,46 +454,47 @@ const respondWithAssetReport = (
   }
 
   const includeInsights = false;
-  client.getAssetReport(assetReportToken, includeInsights, function (
-    error,
-    assetReportGetResponse,
-  ) {
-    if (error != null) {
-      prettyPrintResponse(error);
-      if (error.error_code == 'PRODUCT_NOT_READY') {
-        setTimeout(
-          () =>
-            respondWithAssetReport(
-              --numRetriesRemaining,
-              assetReportToken,
-              client,
-              response,
-            ),
-          1000,
-        );
-        return;
-      }
-
-      return response.json({
-        error,
-      });
-    }
-
-    client.getAssetReportPdf(assetReportToken, function (
-      error,
-      assetReportGetPdfResponse,
-    ) {
+  client.getAssetReport(
+    assetReportToken,
+    includeInsights,
+    function (error, assetReportGetResponse) {
       if (error != null) {
+        prettyPrintResponse(error);
+        if (error.error_code == 'PRODUCT_NOT_READY') {
+          setTimeout(
+            () =>
+              respondWithAssetReport(
+                --numRetriesRemaining,
+                assetReportToken,
+                client,
+                response,
+              ),
+            1000,
+          );
+          return;
+        }
+
         return response.json({
           error,
         });
       }
 
-      response.json({
-        error: null,
-        json: assetReportGetResponse.report,
-        pdf: assetReportGetPdfResponse.buffer.toString('base64'),
-      });
-    });
-  });
+      client.getAssetReportPdf(
+        assetReportToken,
+        function (error, assetReportGetPdfResponse) {
+          if (error != null) {
+            return response.json({
+              error,
+            });
+          }
+
+          response.json({
+            error: null,
+            json: assetReportGetResponse.report,
+            pdf: assetReportGetPdfResponse.buffer.toString('base64'),
+          });
+        },
+      );
+    },
+  );
 };
```

Comment:
are the changes here only due to prettier? or are there any changes? hard to tell

Reply:
I believe these are all due to prettier, cause I didn't make any changes